File: Hard/Array Manipulation/arrayManipulation.py
def arrayManipulation(n, queries):
    arr = [0] * n

    for i in queries:
        arr[i[0] - 1] += i[2]

        if i[1] != len(arr):
            arr[i[1]] -= i[2]
    
    max = 0
    count = 0
    
    for num in arr:
        count += num

        if count > max:
            max = count
    
    return max

File: Hard/Array Manipulation/arrayManipulation.js
function arrayManipulation(n, queries) {
    let arr = []
    for (let i = 0; i < n; i++) arr[i] = 0
    console.log(arr)

    for (let i of queries) {
        arr[i[0] - 1] += i[2]

        if (i[1] != n) {
            arr[i[1]] -= i[2] 
        }
    } 

    let max = 0
    let counter = 0

    for (let num of arr) {
        counter += num

        if(counter > max) {
            max = counter
        }
    }

    return max
}